### DISCUSSION TITLE
Virus Detected with 1.10 Windows Installer

### DISCUSSION THREAD

**Discussion by protons213** [bluenviron/mediamtx#4009] [Questions]
### Question

Does anyone know if something is amiss or can help get the MSI downloaded? Most browsers are blocking the download for me.
Thanks!

**Comment by Taps7734** [comment]
Where did you find an MSI package?

**Comment by protons213** [comment]
Apologies, the ZIP not the MSI.  1.10 comes up as a virus on multiple browsers, 1.9 having no issues.

**Comment by aler9** [comment]
moved in #4012
